bin/build

```diff
@@ -10,41 +10,41 @@ CURRENT_DIR=$("$(dirname "$0")/abspath")
 TOPLEVEL_DIR="$CURRENT_DIR/.."
 KEEP_ALIVE=${KEEP_ALIVE:-}
 
-TAG="$(version_tag)"
+FULL_VERSION_TAG="$(full_version_tag)"
```

Comment:
All of our scripts use similar patterns of using captalized vars for things that are constant-like, exported, or global variables for the script. If we want to switch this, this should be something to discuss in slack outside of this PR probably.

Reply:
@jonahx it's not clear from your comment here, but I think given that this usage is prevalent in our scripts, if this is something that we want to revise we should chat about it and undertake it all at once in a separate effort

Reply:
Resolving this comment to leave only the relevant comments - we can discuss this particular issue in chat if needed

Reply:
@sgnn7 @izgeri so it looks like we're both half-right.  technically (this is documented somewhere, i believe) we're following google bash style guide, which uses all caps for exported vars _and_ constants.  which means, in this case, that, eg, `FULL_VERSION_TAG` is correct but `DOCKER_FLAGS` is not, since that is a variable and not a constant.

I think we should start prefixing the constants with `readonly` as well for extra clarity.